src/renderers/shared/shared/__tests__/ReactCompositeComponentState-test.js

```diff
@@ -345,6 +349,15 @@ describe('ReactCompositeComponent-state', () => {
     expect(() => {
       ReactDOM.unmountComponentAtNode(container);
     }).not.toThrow();
+
+    // Stack doesn't warn
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
My hunch is that `setState` on a parent in `componentWillUnmount` isn't that common but yeah I'll check